utils/ssh/utils.go

```diff
@@ -59,7 +59,7 @@ func fileExist(path string) bool {
 	return err == nil || os.IsExist(err)
 }
 
-func ReadPipe(stdout, stderr io.Reader, alsoToStdout bool) {
+func ReadPipe(stdout, stderr io.Reader, alsoToStdout bool) error {
 	var combineSlice []string
```

Comment:
combineSlice will contains all the cmd output. so we do new combineErrorSlice to gather  error message .